Userland/Applications/ImageViewer/ViewWidget.h

```diff
@@ -1,7 +1,8 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
  * Copyright (c) 2021, Linus Groh <linusg@serenityos.org>
- *
+ * Copyright (c) 2021, Mohsan Ali <mohsan0073@gmail.com>
+ * 
```

Comment:
Extra space at the end of this line
```suggestion
 *
```